use/developer/database-schema/database-cassandra.md

```diff
@@ -0,0 +1,34 @@
+# Database Schema
+
+### 1. Introduction
+At UCI, we leverage Cassandra as our database solution due to its distributed nature and remarkable data access speed. Cassandra's ability to distribute data across nodes enables seamless parallel data access, enhancing both read and write performance. we maintain a comprehensive history of bot conversations using the following schema.
+
+### 2. Table Schema
+
+* Table Name - XMessage
+
+|    Column Name   |     Type           |    Description                                   |
+| ---------------- | ------------------- | ------------------------------------------------ |
+|       id         |        UUID         | Unique identifier for the record.                |
+|    userId        |        Text         | User identifier associated with the message.     |
+|     fromId       |        Text         | Identifier for the sender of the message.        |
```

Comment:
is `formId` correct?

Reply:
Yes this description is correct for fromId.